libs/logger/src/nestjsLoggerServiceAdapter.ts

```
@@ -1,43 +1,53 @@
-export default class NestjsLoggerServiceAdapter
-  extends ConsoleLogger
-  implements LoggerService
-{
+export default class NestjsLoggerServiceAdapter extends ConsoleLogger implements LoggerService {
   public constructor(private readonly logger: Logger) {
     super();
   }
 
-  public info(message: any, ...optionalParams: any[]) : void {
+  public info(message: any, ...optionalParams: any[]): void {
     return this.logger.info(message, this.getLogData(optionalParams));
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)